.travis.yml

```diff
@@ -1,33 +1,61 @@
 language: php
 
-sudo: false
-
 cache:
   directories:
     - $HOME/.composer/cache/files
 
+branches:
+    only:
+        - master
+
 php:
-  - 5.5
-  - 5.6
-  - 7.0
-  - hhvm
+  - '5.5'
+  - '5.6'
+  - '7.0'
+  - '7.1'
+  - nightly
```

Comment:
I removed `hhvm` because it doesn't make much sense IMO: Laravel doesn't test against it so I wouldn't bother with it